setup.py

```diff
@@ -72,13 +72,20 @@ def get_version(file_name: str, version_variable: str = "__version__") -> str:
         "test": ["pytest"],
         "pytorch": ["torch"],
         "tensorflow": ["tensorflow"],
-        # TODO: Add tensorflow to the all target once tf checkpoint support is added.
-        "all": ["torch"],
+        # TODO: Is there a way for all to be the concat of the list of others
+        # instead of making a new list? In case some framework needs multiple
+        # libraries installed?
+        "all": ["torch", "tensorflow"],
     },
     entry_points={
         "git_theta.plugins.checkpoints": [
             "pytorch = git_theta.checkpoints:PickledDictCheckpoint",
             "pickled-dict = git_theta.checkpoints:PickledDictCheckpoint",
+            "tf = git_theta.checkpoints.tensorflow_checkpoint:TensorFlowCheckpoint",
```

Comment:
I guess many tf checkpoint aliases is good for now since users are manually setting the model type at the per-command level with an environment variable. I think we'd discussed moving away from this approach but that's another issue. 

Preliminary I was thinking that we should specify the model type when the model is first tracked with `git theta track <model_glob> --checkpoint-type <checkpoint_type>`. In addition to adding the filter, diff, and merge attributes for the `<model_glob>`, this could also add another attribute called `checkpoint_type`. Then every time we need to get the checkpoint handler, we just need to read the .gitattributes file and pick up the `checkpoint_type` attribute for the glob that matches the current file. The only times we call `get_checkpoint_handler` we always have the working-tree path to the checkpoint file so this should work. Another nice thing about this is that .gitattributes is part of the repo so the checkpoint type will be correctly set for anyone that clones an existing repo.